lib/app/router/routs/wallet/wallet.dart

```diff
@@ -1,7 +1,11 @@
 import 'package:app/app/router/router.dart';
 import 'package:app/feature/add_seed/add_seed_enable_biometry/add_seed_enable_biometry.dart';
 import 'package:app/feature/wallet/wallet.dart';
+import 'package:app/feature/wallet/widgets/account_asset_tab/select_new_asset/select_new_asset.dart';
 import 'package:go_router/go_router.dart';
+import 'package:nekoton_repository/nekoton_repository.dart';
+
+const selectNewAssetAddress = 'selectNewAssetAddress';
```

Comment:
I think we should name it consistently, with 'PathParam' or 'QueryParam' suffix.